src/generate_letters.ml

```diff
@@ -0,0 +1,33 @@
+(** letter_gen is a list containing 7 random chars at any time.
```

Comment:
delete this file, since it has been superseded by `lib/generateLetters.ml`

Reply:
sorry that was deleted i just didnt add those changes will do now